src/model/entities/test/entity-test-factory.ts

```diff
@@ -91,4 +130,12 @@ export class EntityTestFactory {
       ...statusMessage
     }
   }
+
+  public static createActionManifest<ActionManifestData>(actionManifest: Partial<ActionManifest> & { data: ActionManifestData }): ActionManifest<ActionManifestData> {
+    return {
+      actionId: `action-manifest-${process.hrtime.bigint()}`,
```

Comment:
Is this to give it a random id?
If yes, I think there could be a point in streamlining how we do that since we do it in different ways elsewhere.

Reply:
I think, if we can agree that this is a good way to generate unique ID's, then let's make a task for replacing other methods with this one, for later :)

Reply:
It is :) And a very good point indeed. I have created a task in Jira for doing this.